domain/memos/service.go

```diff
@@ -0,0 +1,47 @@
+package memos
+
+import (
+	"errors"
+)
+
+var (
```

Comment:
It seems like a good way to define an exception by scoping it as a global variable in the package! I have a similar thing to use, so I'll have to keep that in mind

ps. how about creating `memos/exception.go`

Reply:
I recommend you read this [moby/errdefs](https://github.com/moby/moby/blob/7b9275c0da/errdefs/defs.go)

Reply:
@devjoeween I also think making another file would be cleaner! I applied your suggestion :) Thanks!

@mcauto I wondered a good error-handling method and their work definitely looks clean. Thank you for recommendation :) I will improve the error handling methods referring to their work :)